pkg/client/clientimplementation/proxy_client.go

```diff
@@ -337,34 +337,47 @@ func (s *proxyClient) ImportWorkspace(ctx context.Context, options client.Import
 	s.m.Lock()
 	defer s.m.Unlock()
 
-	reader, writer := io.Pipe()
-	defer writer.Close()
-	go func() {
-		readLogStream(reader, s.log)
```

Comment:
I think this was needed to have the logs from the command forwarded to the current logging level
So that using the `cli` would have the logs follow along

From what I see this is not going to be the case anymore, as now we only print an error at the end (if any is present) but we lose the rest

Maybe we should find a way to have both to `readLogStream` and to a buffer too, so preserving the old functionality but having also the new (correct) behaviour

Reply:
Ok, I fixed it, now the output is as follows:

```
10:05:45 error could not find corresponding workspace
10:05:45 error Try using the --debug flag to see a more verbose output
10:05:45 fatal error importing a workspace: could not find corresponding workspace
```
So, we have the logs from the child command (could not find corresponding workspace) alongside with the proper error at the end.